exp-models/addon/authenticators/jam-osf-jwt.js

```diff
@@ -23,32 +20,27 @@ export default Base.extend({
                     access_token: accessToken
                 }
             }})
-        }).then(function(res) {
+        }).error(function(/*xhr, status, error*/) {
+            this.transitionToRoute('login');
+        });
+    },
+
+    restore(data) {
+        let accessToken = data.accessToken;
+        return this._post(accessToken).then(function(res) {
             res.data.attributes.accessToken = accessToken;
             return res.data.attributes;
         });
     },
-    authenticate(access_token, _ /* expires */) {
-        return Em.$.ajax({
-            method: 'POST',
-            url: this.url,
-            dataType: 'json',
-            contentType: 'application/json',
-            xhrFields: {withCredentials: true},
-            data: JSON.stringify({data: {
-                type: 'users',
-                attributes: {
-                    provider: 'osf',
-                    access_token
-                }
-            }})
-        }).then(function(res) {
-            res.data.attributes.accessToken = access_token;
+    authenticate(accessToken, _ /* expires */) {
```

Comment:
Why is there an underscore in this call signature? Can the argument simply be omitted to ignore it instead?

As a bonus, that would get rid of another jshint error:
`exp-models/addon/authenticators/jam-osf-jwt.js: line 35, col 31, '_' is defined but never used.`